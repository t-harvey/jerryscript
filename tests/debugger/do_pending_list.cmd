fbreak do_pending_breakpoints.js:50
fbreak do_pending_breakpoints.js:235532
fbreak do_pending_breakpoints.js:2145
n
list pending
c
