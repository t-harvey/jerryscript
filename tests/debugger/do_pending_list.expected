Connecting to: localhost:5001
Stopped at tests/debugger/do_pending_list.js:15
(jerry-debugger) fbreak do_pending_breakpoints.js:50
Breakpoint not found
Pending breakpoint added
(jerry-debugger) fbreak do_pending_breakpoints.js:235532
Breakpoint not found
Pending breakpoint added
(jerry-debugger) fbreak do_pending_breakpoints.js:2145
Breakpoint not found
Pending breakpoint added
(jerry-debugger) n
Stopped at tests/debugger/do_pending_list.js:17
(jerry-debugger) list pending
0: 50
1: 235532
2: 2145
(jerry-debugger) c
